method/seurat/seurat_optargs.json

```diff
@@ -0,0 +1,7 @@
+{
+    "matrix": "dimensionality_reduction",
+    "integrated_feature_selection": false,
+    "image": false,
+    "neighbors": false,
```

Comment:
Can we use existing neighbor graphs for seurat implementation? Likely seurat-based SNN method will be included in centralized graph building.

Reply:
I don't think we want that. Seurat and scanpy are supposed to be the "non-spatial" control. The neighbor graph generation that we are going to look at will be based on spatial information. Seurat and scanpy would usually use neigbors in the gene expression space.